packages/big-design/src/components/Form/__snapshots__/spec.tsx.snap

```diff
@@ -166,7 +166,7 @@ exports[`simple form render 1`] = `
 
 @media (min-width:720px) {
   .c0 .c6,
-  .c0 .styled__StyledTextareaWrapper-c1uos0-0 {
+  .c0 .styled__StyledTextareaWrapper-sc-c1uos0-0 {
```

Comment:
The new `babel-plugin-styled-components` version prefixes styled-components with `sc` now.